content/api/trafiklab-apis/netex-regional/siri.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Siri
+aliases:
+- "api/netex-regional-siri"
+weight: 20 
+date: 2022-04-14
+---
+
+Siri is service Interface for Real Time Information.
+
+The Norwegian profile includes Estimated Timetables (ET), Situation Exchange (SX) and Vehicle Monitoring (VM) data.
+These are consistent with GTFS-RT TripUpdates, ServiceAlerts and VehiclePositions.
+
+
+{{% note %}}
+**Where to download**
+
+The dataset can be accessed through the following URL:
+
+- [https://opendata.samtrafiken.se/siri/siri.pb?key={apikey}](https://opendata.samtrafiken.se/siri/siri.pb?key={apikey})
+
+Replace `{apikey}` with your own API key.
+{{% /note %}}
+
```

Comment:
Add an info block 

"Availability of regional data differs per operator. See the overview page to see which data is provided by the operator(s) you are interested in." similar to the one for GTFS Regional Realtime